Scripts/Scratchpad/Extensions/keypad.lua

```diff
@@ -1,41 +1,36 @@
 -- A minimal keypad meant for coordinate input
 
-local x = 0
-local function advance(w)
-  local before = x
-	x = x + w
-	return before
-end
+local keyMatrix = {
+  { "7", "8", "9", "N", "S" },
+  { "4", "5", "6", "W", "O" },
```

Comment:
O should be E ?

Reply:
Oops, yes! O is German _Ost_ 🙈 - will fix it

Reply:
Ahhh that makes sense!